### ISSUE TITLE
Feature rquest: enable deploying lambda with container image

### ISSUE THREAD

**Issue by aaron-c-zhao** [awslabs/aws-deployment-framework#398]
Hi there,

### Feature
It would be nice to have support for using a container image as the deployment package of Lambda.

### Use case
Sometimes we use heavy libraries in our Lambda that the 250MB size limit of a  .zip package simply does not suffice. Therefore, a container image that can go up to 10GB would help. 

### Current workaround
Now, we are manually modifying *package_transfrom.sh* and add *--image-repository* argument to *sam package*

**Comment by StewartW** [comment]
Hi, can you share some more information about this use case please?  Are you calling package_transform from your buildspec.yml?

**Comment by aaron-c-zhao** [comment]
> Hi, can you share some more information about this use case please? Are you calling package_transform from your buildspec.yml?

Indeed, we keep the modified package_transform.sh locally, and call it from buildspec.yml. 

Please let me know if any more specific information would help.
